v2/interface.go

```diff
@@ -61,6 +61,8 @@ type ClientConfiguration struct {
 	// any request parameters or request or response fields that correspond to
 	// alpha features.
 	EnableAlphaFeatures bool
+	// CAData holds PEM-encoded bytes (typically read from a root certificates bundle).
```

Comment:
I have changed the way that the cert is added to TLSClientConfig.RootCAs so that any cert supplied in ClientConfiguration.CAData will be appended to the root CAs rather than overwriting any root CAs. I have also added a comment to that effect on the ClientConfiguration.CAData field.